public/static/docs/command-reference/repro.md

```diff
@@ -43,7 +43,8 @@ before running the stages that produce them.
 `dvc repro` does not run `dvc fetch`, `dvc pull` or `dvc checkout` to get data
 files, intermediate or final results. It saves all the data files, intermediate
 or final results into the <abbr>DVC cache</abbr> (unless `--no-commit` option is
-specified), and updates stage files with the new checksum information.
+specified), and updates stage files with the new dependency/output file or
```

Comment:
structure is complicated  "updates hash values for dependencies and outputs in a stage file"

Reply:
Yeah actually this whole paragraph wasn't completely correct I think. Please review again, updated in cb134fe2.

Reply:
and 5fbec22